template/node/src/rpc.rs

```diff
@@ -188,20 +188,19 @@ where
 	}
 
 	io.merge(
-		Net::new(
+		EthPubSub::new(
+			pool,
 			client.clone(),
 			network.clone(),
-			// Whether to format the `peer_count` response as Hex (default) or not.
-			true,
+			subscription_task_executor,
+			overrides,
 		)
 		.into_rpc(),
 	)?;
 
-	io.merge(Web3::new(client.clone()).into_rpc())?;
+	io.merge(Net::new(client.clone(), network, true).into_rpc())?;
```

Comment:
A nit -- can you add back the comment originally there for the third argument?

Reply:
Done